### ISSUE TITLE
[bug] iOS & Voiceover: cannot select login

### ISSUE THREAD

**Issue by webberian** [kolplattformen/skolplattformen#482]
Since version 2.0.0 (?) of the iOS app, it has not been possible via Voiceover to select different elements of the start/login page. Voiceover can read all the text on the page, but neither the language control nor the login dropdown element is selectable.

iPhone Xr iOS 14.8

On Android with Talkback, this is not a problem and language and login controls can be selected and activated.
